### PR TITLE
Review Page Partial Setup

### PR DESCRIPTION
**Pull request by chrisj117**
**_WHAT DOES THIS PR DO?_**

  - This PR adds features that will be used in the final Review page (Review.jsx). These include:
     - A review button
     - An additional comments box
     - A professor drop down menu
  - Full functionality, element positioning, and example data are not finalized
  - To access the Review page, a temporary alteration was made to the page routing (main.jsx)

**_WHAT ISSUES ARE RELATED TO THIS PR?_**

  - #5 
  - #7 
  - #10
  - #43

**_HOW DO I TEST OUT THIS PR?_**

  - Run this project locally
  - Navigate to the "Review" page by going to http://localhost:5173/review
  - Upon loading, the following page should appear:
image

### CODE REVIEW

File: api/backend/settings.py
Comment (danielogen) [inline api/backend/settings.py:1]: Hi @chrisj117 , I don't think this change should appear in this PR :)

**Review by danielogen — CHANGES_REQUESTED** [review]
Some backend file was touched... This shouldn't happen based on our current development workflow

File: api/backend/settings.py
Comment (chrisj117) [reply api/backend/settings.py:1]: You're correct, not sure what's going on with the file diff in settings.py, I'll put this PR back into a draft until I can clear this up

**Review by azboss2021 — APPROVED** [review]
Looks good @chrisj117

**Review by tresimovic — APPROVED** [review]
Looks good

**Review by carrec-2 — APPROVED** [review]
Keep up the good work!

File: api/backend/settings.py
Comment (chrisj117) [reply api/backend/settings.py:1]: Seems the file diff was no newline at EOF, can't seem to remove it but it should not cause problems

**Review by danielogen — APPROVED** [review]
Requested changes has been effected. Thank you!

**Pull request merged**